### DISCUSSION TITLE
gRPC-Core 1.44.0 warnings

### DISCUSSION THREAD

**Discussion by winklerchristian123** [firebase/firebase-ios-sdk#9700] [Q&A] [answered]
Dear All,

I have a question, since I updated to the latest version of gRPC-Core 1.44.0-grpc I get several hundred warnings regarding the same issue.

Have you any hint how to fix this?

Thanks & Regrads,
![warnings](https://user-images.githubusercontent.com/98278941/165045416-ace64142-2e11-41e7-bb07-cb9c3363ed52.jpg)

**Comment by paulb777** [comment, marked as answer]
Hi @crazysnake666 Thanks for the report. This is an issue in Firestore's gRPC dependency and we've asked the gRPC team to take a look.

cc: @dennycd @wu-hui

**Reply by paulb777** [reply]
The issue is being tracked at #9790. Please follow it there.
